resend/exceptions.py

```diff
@@ -172,6 +171,16 @@ def __init__(
 
 
 def raise_for_code_and_type(code: str, error_type: str, message: str) -> ResendError:
+    """Raise the appropriate error based on the code and type.
+
+    Args:
+        code: The error code
+        error_type: The error type
+        message: The error message
+
+    Returns:
+        ResendError: The error object
```

Comment:
can we say here that it will raise an error from the ERROR_LIST or defaults to ResendError ?

Reply:
absolutely, changing it